cmd/k8s-operator/deploy/manifests/proxy.yaml

```diff
@@ -14,10 +14,8 @@ spec:
         - name: sysctler
           securityContext:
             privileged: true
-          command: ["/bin/sh"]
-          args:
-            - -c
-            - sysctl -w net.ipv4.ip_forward=1 net.ipv6.conf.all.forwarding=1
+          command: ["/bin/sh", "-c"]
+          args: [sysctl -w net.ipv4.ip_forward=1; if sysctl net.ipv6.conf.all.forwarding; then sysctl -w net.ipv6.conf.all.forwarding=1; fi]
```

Comment:
I think(?) using `;` instead of `&&` here means that if the first `sysctl` fails but the second one succeeds, it will look to us as if the whole thing succeeded. Is that what we want?